### ISSUE TITLE
Function to move file from module to mysite?

### ISSUE THREAD

**Issue by sanderha** [unclecheese/LangEditor#11]
This module looks awesome as hell. It just seems weird that when you copy a lang file to a new language, it ends up in that selected module, and therefore would be deleted when the project is installed somewhere else, because the modules are repo's. 

Are you supposed to exclude all modules from the list and then created a lang file in mysite for everything, which users can then copy?

**Comment by unclecheese** [comment]
Yeah, that's precisely why this module is not awesome as hell. I wouldn't use it. It enforces a major anti-pattern, as you've pointed out.
